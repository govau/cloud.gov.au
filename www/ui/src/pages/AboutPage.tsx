import * as React from "react";

interface Props {}

const AboutPage: React.StatelessComponent<Props> = () => (
  <React.Fragment>
    <h2>About cloud.gov.au</h2>
    <p>This page is a placeholder.</p>
  </React.Fragment>
);

export default AboutPage;
