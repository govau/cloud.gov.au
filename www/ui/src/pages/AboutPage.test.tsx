import * as React from "react";
import * as renderer from "react-test-renderer";

import AboutPage from "./AboutPage";

it("renders without crashing", () => {
  const tree = renderer.create(<AboutPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
