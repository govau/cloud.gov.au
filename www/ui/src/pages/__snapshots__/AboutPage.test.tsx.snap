// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Array [
  <h2>
    About cloud.gov.au
</h2>,
  <p>
    This page is a placeholder.
</p>,
]
`;
